solutions/hw1/task2/bash.py

```diff
@@ -0,0 +1,69 @@
+import os
+
+
+def wc(first: str, *args: str):
+    def wc_for_one(filepath: str) -> dict:
```

Comment:
`dict` is very broad, because anything can be in the dictionary. You should use `Dict` from `typing`, and here it will be `Dict[str, int]`. Then the user immediately knows what's in the keys and values.